contact-form

muneeb3123 opened the pull request:
This PR includes:
Completed the mobile version of the portfolio using HTML and CSS
Implemented linters to ensure code quality and consistency
Tested and ensured responsiveness according to the requirement
Integrated with existing portfolio design for a cohesive user experience.

muneeb3123 committed:
Add contact form

muneeb3123 committed:
style the form section

xandro2021 committed:
Fix Background Image

+ Add new little image instead of a full background
+ Positioned those images relative to the container
+ Use pseudo-clases to add those icons as part of the background

muneeb3123 committed:
add max length of for message

xandro2021 committed:
Add second author in the README.md

muneeb3123 committed:
Update README.md

update Readme.md

Alfredbis29 commented on style.css:
```diff
@@ -404,3 +404,118 @@ nav #mylogo {
+.btn:active {
+  background-color: #2230d2;
+  color: #fff;
+}
```
- [ ] Nice Design so far but something need to be improved for your project to look professional , I just noticed that your container didn't fit all the screen , your mobile version should be responsive , kindly fix this in the video I presented below :upside_down_face:  :point_down: 

https://user-images.githubusercontent.com/88894525/220473258-4795a002-532d-464c-b31a-9e76ef700145.mp4

muneeb3123 replied:
@Alfredbis29 this video has no sound .will you explain it  well?

Alfredbis29 commented on index.html:
```diff
@@ -10,189 +11,223 @@
+          <button class="btn" type="submit" >Get in Touch</button>
+        </form>
+      </section>
 
```
- [ ] You did a great job so far on this project , but you forget to add the inline bar below your get in touch bouton , kindly add for your project to match the figma design and be more attractive :wink:  try to fix the font on your design and make it as we can see on the figma design :smile: 

Your Design :point_down: 

![Screenshot from 2023-02-22 00-31-45](https://user-images.githubusercontent.com/88894525/220474422-066794f3-4bca-4f6c-a221-8c7082448453.png)

Figma Design :point_down: 
![figmalinebar](https://user-images.githubusercontent.com/88894525/220474524-d7588c99-e0eb-4b27-a2fe-e524118d5d0b.png)

Alfredbis29 reviewed (changes requested):
### Hi @muneeb3123   @xandro2021 ,
 You did a great job so far 👏🏼 👏🏼 There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/ea601010ed4ac26655ecf8ef2441aea1c7d8b3ab5cc23c495c981ea928fb2043/68747470733a2f2f6d65646961342e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) 
 
 ### To highlight 💪🏻
Your project is professional :white_check_mark:
Your repo is okay :white_check_mark:
No Linter errors :white_check_mark: 
:+1:  Good job so far!

## STATUS: CHANGES REQUESTED 

- [x] _Check under the review

 
## Optional suggestions

- [ ] KIndly follow the project requirement and I'm sure you are going to make it :wink: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 Cheers and Happy coding! :clap: :clap: :clap: :computer: Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please ping me @alfredbis29 when you comment so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

xandro2021 committed:
Fix horizontal scrolling and add bottom line

+ Resize the background icons according with the codereview suggestion
+ Add the line icon at the bottom of home page
+ Cetered and resized the line icon at the bottom
+ Resize the title in the form that was causing the horizontal scrolling issue

Zuhaib042 commented on style.css:
```diff
@@ -404,3 +404,124 @@ nav #mylogo {
+  top: 1%;
+}
+
+.container::after {
+  content: "";
+  display: block;
+  width: 19rem;
+  height: 9rem;
+  background-image: url("./images/icon03.png");
+  background-size: 5rem;
+  background-repeat: no-repeat;
+  background-position: 100% 99%;
+  position: absolute;
+  right: 15%;
+  bottom: 9%;
+}
```
- [ ] Good job implementing the suggested changes by the previous reviewer but there is another issue.
these lines of code that add the decoration images are somehow blocking your form submit button to function properly. kindly either consider removing them and then consider downloading the whole background decoration image from the figma and then placing it in the form section as a background image, or you can adjust it as it is now and make sure that your form submission button is working.

xandro2021 replied:
+ He can also add the property **z-index: 1** to the children components  

z-index: 0 is the default value of all the components.
The pseudo-class is blocking the children's components because of having the same level as z-index.
So in order to avoid playing too much with the width, height and background-size to avoid blocking issues. It is easier to give all the children components of the container a z-index: 1 so they would not be blocked.



    .container > * {
      z-index: 1;
    }

Zuhaib042 reviewed (changes requested):
Hi @muneeb3123 @xandro2021 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![close](https://media.giphy.com/media/3o7qDQAFlF8mZjv6fK/giphy.gif)

 ## Highlights
✅ Good job implementing the requested changes

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

xandro2021 committed:
Avoid the form being blocked by the background elements

+ Just move everything else a level up by increasing its z-index

binhussen reviewed (approved):
Hi @muneeb3123 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
